tests/e2e/matmul/CMakeLists.txt

```diff
@@ -2251,6 +2347,31 @@ iree_generated_e2e_runner_test(
     "requires-gpu-cdna3"
 )
 
+iree_generated_e2e_runner_test(
+  NAME
+    e2e_matmul_cdna_experimental_dt_f32_f32
+  TEST_TYPE
+    matmul
+  GENERATOR
+    "generate_e2e_matmul_tests.py"
+  GENERATOR_ARGS
+    "--lhs_rhs_type=f32"
+    "--acc_type=f32"
+    "--shapes=small"
+  TEST_RUNNER
+    iree_tools_testing_e2e_iree-e2e-matmul-test
+  TARGET_BACKENDS
+    "rocm"
+  DRIVERS
+    "hip"
+  COMPILER_FLAGS
+    ${IREE_HIP_TEST_COMPILER_FLAGS}
+    "--iree-opt-data-tiling"
+    "--iree-global-opt-enable-early-materialization=false"
+  LABELS
+
```

Comment:
Do we need to add some labels? Perhaps @ScottTodd can answer it?

```
    "noasan"
    "nomsan"
    "notsan"
    "noubsan"
    "requires-gpu-cdna3"
```

Reply:
Match what others in the file are doing for now.

The sanitizer tags are mostly a relic of the downstream Google build system and I've been thinking about removing them. Here on GitHub we currently only run CPU tests with sanitizers enabled, so the labels would only affect developers manually running GPU tests + ASan/TSan/MSan. At the point where you are running an explicit config like that you might as well add another explicit filter to your build/test command, so the labels aren't really useful IMO.

Reply:
> Match what others in the file are doing for now.

done.